keyboards/preonic/keymaps/laurentlaurent/keymap.c

```diff
@@ -0,0 +1,603 @@
+/* Laurent's Preonic Layout
+ */
+
+// For IntelliSense
+    #ifdef __INTELLISENSE__
+        #include "../../rev3/config.h"
+        #include "../../rev3/rev3.h"
+        enum dynamic_macro_keycodes {
+            DYN_REC_START1 = DYNAMIC_MACRO_RANGE,
+            DYN_REC_START2,
+            DYN_REC_STOP,
+            DYN_MACRO_PLAY1,
+            DYN_MACRO_PLAY2,
+        };
+        #define QMK_KEYBOARD_H "config.h"
```

Comment:
This would be wrong, actually.  It would be the "rev3.h", not the config.h.

Reply:
Fixed! Thanks